react/src/index.css

```
@@ -20,4 +20,13 @@ body,
     justify-content: center;
     align-items: center;
     height: 60px;
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+    width: 350px;
```

```suggestion
    width: 100%;
    max-width: 600px;
```
Suggestion: so other, larger components fit in and we can play a bit with responsiveness.